### PR TITLE
doc fix - set method is also required

### PR DESCRIPTION
**Pull request by vitorreis**
In order to create a new configuration class you need to provide also a setter for HttpClientConfiguration

### CODE REVIEW

File: docs/source/manual/client.rst [hunk 1]
```diff
@@ -34,6 +34,10 @@ To create a :ref:`managed <man-core-managed>`, instrumented ``HttpClient`` insta
         public HttpClientConfiguration getHttpClientConfiguration() {
             return httpClient;
         }
+        
+        public void setHttpClientConfiguration(HttpClientConfiguration httpClient) {
+            this.httpClient= httpClient;
```
Comment (joschi) [inline docs/source/manual/client.rst]: Missing space before the equals sign.

File: docs/source/manual/client.rst [hunk 2]
```diff
@@ -34,6 +34,10 @@ To create a :ref:`managed <man-core-managed>`, instrumented ``HttpClient`` insta
         public HttpClientConfiguration getHttpClientConfiguration() {
             return httpClient;
         }
+        
+        public void setHttpClientConfiguration(HttpClientConfiguration httpClient) {
```
Comment (joschi) [inline docs/source/manual/client.rst]: For the sake of completeness this should also be annotated with `@JsonProperty("httpClient")`.

Comment (vitorreis) [inline docs/source/manual/client.rst]: Indeed. fixed.

File: docs/source/manual/client.rst (hunk 1 above)
Comment (vitorreis) [inline docs/source/manual/client.rst]: Indeed. fixed.

**Comment by joschi** [comment]
Thanks for your PR!

**Comment by vitorreis** [comment]
@joschi Happy to help =)

**Pull request merged**